common/expression/src/chunk.rs

```diff
@@ -12,29 +12,69 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use itertools::Itertools;
-
-use crate::values::Column;
+use crate::types::AnyType;
+use crate::Value;
 
+/// Chunk is a lightweight container for a group of values.
 pub struct Chunk {
-    columns: Vec<Column>,
+    values: Vec<Value<AnyType>>,
```

Comment:
I think `columns` as the field name may sound more reasonable in the context of `Chunk`.

Reply:
Because we should care about the constant column, which is memory saved representation than column.